examples/using-JSON-YAML-source/.prettierrc

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
You can get rid of this file, since the Prettier config from the larger project can be used instead.